questionnaire/questionnaire-service.js

```diff
@@ -3,7 +3,7 @@
 const service = require('./request-service')();
 
 function questionnaireService() {
-    function createQuestionnaire() {
+    async function createQuestionnaire(userId = undefined) {
```

Comment:
update to:

````js
async function createQuestionnaire({userId}) {
````

variables are inherently `undefined`, no need to explicitly set them as that